keyboards/adkb96/keymaps/default/keymap.c

```diff
@@ -0,0 +1,23 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define BASE 0
```

Comment:
I would recommend either converting these layer names to an enum, or removing them entirely, since there is only one layer here.